Homework2/Task1/Pasha_Vinnikov/Variables

```diff
@@ -0,0 +1,20 @@
+ numberOfShips 		    Количество кораблей в порту
```

Comment:
numbersOfShips or shipsNomberCount

Reply:
As far as I understand the expression "the nunmber of ships" we can interpret like the quantity or amount of ships. So I thik it's all right with this string.